paddle/fluid/framework/data_feed.cc

```diff
@@ -1427,21 +1435,14 @@ int PaddleBoxDataFeed::GetCurrentPhase() {
 
 void PaddleBoxDataFeed::PutToFeedVec(const std::vector<Record*>& ins_vec) {
 #ifdef _LINUX
-  std::vector<std::vector<float>> batch_float_feasigns(use_slots_.size(),
-                                                       std::vector<float>());
-  std::vector<std::vector<uint64_t>> batch_uint64_feasigns(
-      use_slots_.size(), std::vector<uint64_t>());
-  std::vector<std::vector<size_t>> offset(use_slots_.size(),
-                                          std::vector<size_t>{0});
-  std::vector<bool> visit(use_slots_.size(), false);
-  ins_content_vec_.clear();
-  ins_content_vec_.reserve(ins_vec.size());
-  ins_id_vec_.clear();
-  ins_id_vec_.reserve(ins_vec.size());
```

Comment:
where is the corresponding update to `ins_content_vec` and `ins_id_vec`?

Reply:
remove this for code consistency